psamm/datasource/sbml.py

```diff
@@ -872,12 +894,20 @@ def write_model(self, file, model):
             species_tag.set(
                 self._sbml_tag('compartment'),
                 model_compartments[species.compartment])
+            species_tag.set(self._sbml_tag('constant'), 'true')
+            species_tag.set(self._sbml_tag('boundaryCondition'), 'true')
+            species_tag.set(self._sbml_tag('hasOnlySubstanceUnits'), 'true')
+
+        if self._cobra_flux_bounds is False:
```

Comment:
As long as the old and new parameters are consistent I don't think it will be an issue. SBML files are not usually edited directly by users. Also, the default should be to generate only the FBC parameter list and not the local parameters.